node/tests/RedisClientInternals.test.ts

```diff
@@ -307,17 +307,19 @@ describe("SocketConnectionInternals", () => {
                     );
                 });
                 const result = await connection.get("foo");
-                expect(result).toEqual(expected);
+                expect(intoString(result)).toEqual(intoString(expected));
             });
         };
 
         it("should pass strings received from socket", async () => {
             await test_receiving_value("bar");
         });
 
-        it("should pass maps received from socket", async () => {
-            await test_receiving_value({ foo: "bar", bar: "baz" });
-        });
+        // TODO: Because the maps are not arranged consistently, it is difficult to compare them.
```

Comment:
I did fix this in `intoString` for `Set`, a similar solution can be used for `Map`:

For `Set` I am sorting the array before serializing it:

```typescript
    } else if (obj instanceof Set) {
        const arr = Array.from(obj);
        arr.sort();

        for (const item of arr) {
            intoArrayInternal(item, builder);
        }
    } else if (obj instanceof Map) {
```

Reply:
Sort was added to the Map case and to the last else case, I guess it for the ReturnTypeMap type. The test uses this type.